lib/flight_scheduler/schedulers/fifo_scheduler.rb

```diff
@@ -69,6 +69,13 @@ def run_allocation_loop(candidates)
       .select { |job| job.reason_pending == 'WaitingForScheduling' }
       .each { |job| job.reason_pending = 'Priority' }
 
-    new_allocations
+    excess = partition.nodes.any? { |n| n.state == 'IDLE'}
+    [new_allocations, excess]
+  end
+
+  def excess_resources?(partition)
+    FlightScheduler.app.job_registry.jobs.any? do |job|
+      job.partition == partition && job.reason_pending == 'Resources'
+    end
   end
```

Comment:
What's this method for?  It doesn't seem to be called from anywhere.

Reply:
It's junk from an implementation I scrapped. Removed: https://github.com/openflighthpc/flight-scheduler-controller/pull/42/commits/fa669934fc0b336f566044a9fa1fee62dedc5caa